pkgs/development/libraries/essentia/default.nix

```diff
@@ -0,0 +1,83 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, fetchpatch
+, libyaml
+, fftwFloat
+, ffmpeg
+, chromaprint
+, taglib
+, libsamplerate
+, pkgconfig
+# This one is not listed in upstream's instructions but it seems to be a
+# requirement since the build fails with the error:
+# [290/290] Linking build/src/libessentia.so
+# /nix/store/cl1i6bfqnx48ipakj4px7pb1babzs23j-binutils-2.31.1/bin/ld: cannot find -lQtCore
+# Probably because gaia which depends on qt4
+, qt4
+, wafHook
+, pythonSupport ? true
+, python
+, gaiaSupport ? true
+, gaia
+, withVamp ? true
+, withExamples ? true
+}:
+
+assert pythonSupport -> python != null;
+assert gaiaSupport -> gaia != null;
+
+stdenv.mkDerivation rec {
+  pname = "essentia";
+  # other versions fail to build
+  version = "2.1_beta5";
+
+  src = fetchFromGitHub {
+    owner = "MTG";
+    repo = "essentia";
+    rev = "v${version}";
+    sha256 = "0ig267wfxry2rjpij4fypf2zk7f59jd0n23l3807igbn2cmkgz4w";
+  };
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/MTG/essentia/pull/915.patch";
```

Comment:
Please make sure to add a comment (eg. `Make waf use ${PREFIX} to install examples` - the title of your pull request) about what this patch does because it's not clear from the filename.

Reply:
:+1: